examples/using-gatsby-source-graphql/package.json

```diff
@@ -22,7 +22,6 @@
     "eslint": "^4.19.1",
     "eslint-config-google": "^0.9.1",
     "eslint-config-prettier": "^2.9.0",
-    "eslint-plugin-flow-vars": "^0.5.0",
```

Comment:
I'm really not sure why _any_ of these eslint dependencies are in this example, since it doesn't seem to be used whatsoever, but oh well!